### PR TITLE
feat: Statuspage

### PR DESCRIPTION
**Pull request by MattIPv4**
Add the Statuspage embed script to the site, which will automatically show a notification in the bottom right of the screen if there is an active incident or maintenance on the status page site. This also adds a link in the footer to the status page.

image

I don't expect this to be merged for a while, access, policies and more for the status page need to be established first (see https://github.com/nodejs/build/issues/2265 & https://github.com/nodejs/build/pull/2299).

### CODE REVIEW

**Comment by nschonni** [comment]
Personally, I would skip the script and just include the link

**Comment by MattIPv4** [comment]
Imo, this whole project of adding a status page is about better communication with the community when incidents occur. The script enables the embed, which when an incident is active, will show a nice obvious alert on the site to massively increase visibility and awareness of the ongoing issue. Having just a link to the status page in the footer does very little to help with effectively communicating incidents to the community.

**Comment by nschonni** [comment]
🤷 that's what GitHub and CloudFlare seem to do as well

File: layouts/partials/footer.hbs [hunk 1]
```diff
@@ -12,6 +12,7 @@
           <li><a href="https://github.com/nodejs/node/issues">{{site.reportNodeIssue}}</a></li>
           <li><a href="https://github.com/nodejs/nodejs.org/issues">{{site.reportWebsiteIssue}}</a></li>
           <li><a href="https://github.com/nodejs/help/issues">{{site.getHelpIssue}}</a></li>
+          <li><a href="https://status.nodejs.org">{{site.statusPage}}</a></li>
```
Comment (XhmikosR) [inline layouts/partials/footer.hbs]:
```suggestion
          <li><a href="https://status.nodejs.org/">{{site.statusPage}}</a></li>
```

**Comment by Trott** [comment]
As much as I'd love to have the feature, I'm going to close this as stale. Feel 100% free to re-open.

**Pull request closed without merging**